src/content/developers/docs/mev/index.md

```diff
@@ -116,6 +116,92 @@ While many searchers are still making good money from MEV, as opportunities beco
 
 As DeFi grows and increases in popularity, MEV may soon significantly outweigh the base Ethereum block reward. With that comes a growing possibility of selfish block remining and consensus instability. Some consider this to be an existential threat to Ethereum, and disincentivizing selfish mining is an active area of research in Ethereum protocol theory. One solution currently being explored is [MEV reward smoothing](https://ethresear.ch/t/committee-driven-mev-smoothing/10408).
 
+## MEV after The Merge {#mev-after-the-merge} 
```

Comment:
Feel like this wording worked when TM was still looming --now it is in the past maybe we refer to "MEV since The Merge" or "PoS-specific MEV" or similar?

Reply:
Does "MEV in Ethereum Proof-of-Stake (PoS)" sound good? I like "MEV since the Merge", but it may not be ideal for SEO.